src/Components/CreateImageWizard/CreateImageWizard.scss

```diff
@@ -97,5 +103,16 @@ div.pf-v5-c-alert.pf-m-inline.pf-m-plain.pf-m-warning {
     // flex: 1 1 0
     
     // TO DO: Remove after the fix
-    min-height: 800px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    min-height: 800px;  /* Always go with pixels */
+}
+
+.pf-v5-c-wizard__footer {
+    position: sticky;
```

Comment:
I'm not sure I get how this works with flex. But sounds like we can do one or the other, or how does this work? Can you elaborate a bit pls? I'm bit lost in css positioning :thinking: